runtime/v1/shim/service.go

```diff
@@ -511,33 +511,35 @@ func (s *Service) allProcesses() []rproc.Process {
 }
 
 func (s *Service) checkProcesses(e runc.Exit) {
-	shouldKillAll, err := shouldKillAllOnExit(s.bundle)
```

Comment:
`s.bundle` might be nil when the `Create` doesn't update `s.bundle`. `shouldKillAllOnExit` should be called for init process only. Therefore, it can be prevent data race